package.json

```diff
@@ -201,6 +202,11 @@
           "when": "resourceFilename == sdkconfig",
           "command": "menuconfig.start",
           "group": "navigation"
+        },
+        {
+          "when": "resourceFilename == partition-table.bin || resourceExtname == .csv",
```

Comment:
Question, why is the editor activated on partition-table.bin? Isn't it just the build artifact?

Reply:
Yes Initially I thought that will allow user to edit the bin files directly as well (`bin` -> `csv` -> `edit` -> `csv` -> `.bin` -> `flash`) ...but will remove this functionality because will create a race condition between `normal build` and `partition table editing the bin`